DEVELOPER.md

```diff
@@ -1,5 +1,48 @@
 # Developer Notes
 
+## Setup your development environment
+
+First fork https://github.com/m3db/m3 into your own workspace on Github.
+
+Then create your clone:
+
+```bash
+export working_dir=$GOPATH/src/github.com/m3db
+mkdir -p $workding_dir
+
+# Set this to your Github user
+export user={your github profile name}
+
+# Clone your fork
+cd $working_dir
+git clone git@github.com:$user/m3.git
+# or: https://github.com/$user/m3.git
+
+# Set upstream
+git remote add upstream git@github.com:m3db/m3.git
```

Comment:
Perhaps `cd` into `${working_dir}/m3` first?

Reply:
See line 17 :)

Reply:
You're right of coure